config/routes.rb

```diff
@@ -0,0 +1,14 @@
+Rails.application.routes.draw do
+  root to: 'homepages#index'
+
+  resources :works
```

Comment:
Weird that your edit page for works isn't working. I'm going to take a peek at that page too!